server/plugin/command.go

```diff
@@ -589,26 +598,31 @@ func getAutocompleteData(config *Configuration) *model.AutocompleteData {
 	mute.AddCommand(muteList)
 
 	settings := model.NewAutocompleteData("settings", "[setting] [value]", "Update your user settings")
-	setting := []model.AutocompleteListItem{{
-		HelpText: "Turn notifications on/off",
-		Item:     "notifications",
+	settingNotifications := model.NewAutocompleteData("notifications", "", "Turn notifications on/off")
+	settingValue := []model.AutocompleteListItem{{
+		HelpText: "Turn setting on",
```

Comment:
Given that there is now a per settings list of option: Should the help texts be more specific i.e.
```suggestion
		HelpText: "Turn notifications on",
```

Reply:
resolved , text are more specific